src/components/KButton/KButton.vue

```diff
@@ -282,6 +308,20 @@ export default {
     }
   }
 
+  // reset the style of a button to bare minimum
+  // using :where() here to make style override easier
+  &:where(.none) {
```

Comment:
I still think a `disabled` button with `appearance="none"` needs to have the disabled style applied whereby it looks "greyed out" etc.

![image](https://github.com/user-attachments/assets/ec895216-a794-4ff0-83f4-337ba6792afa)

The most common use case we have currently for `appearance="none"` is a button wrapping an icon, so we'd still want the disabled styles here.

Reply:
I see. What I was trying to do is create a `<button>` that looks like a plain `<div>` but still aligns with our core functionality and behaviors. If we want it to be styled out-of-the-box maybe we should take another approach? In my previous design language project we have text buttons and icon buttons, which can be used inside Table rows or as close buttons for Dialogs.

Reply:
I think the only exception would be here for `disabled` -- removing all other styles (e.g. size, etc.) is what I would expect.

Reply:
We also still want the `cursor: pointer` on `appearance="none"` as well if it's not there already

Reply:
...and maybe the border radius for the `focus-visible` state 😬

Reply:
Added `border-radius` (for other appearances it's related to `size` but here I just add a medium sized one) and disabled color.